CHANGELOG.md

```diff
@@ -1,5 +1,132 @@
+* Added error check for extra quotation for extra-config. [#10886](https://github.com/kubernetes/minikube/pull/10886)
+* Fix the failure of `minikube mount` in case of KVM2 [#10733](https://github.com/kubernetes/minikube/pull/10733)
+* Fix/minikube status for scheduled stop [#10911](https://github.com/kubernetes/minikube/pull/10911)
+* create network: use locks and reservations to solve race condition [#10858](https://github.com/kubernetes/minikube/pull/10858)
+* fix driver.IndexFromMachineName() [#10821](https://github.com/kubernetes/minikube/pull/10821)
+* Fix link for pushing images docs on cache cmd [#10880](https://github.com/kubernetes/minikube/pull/10880)
+* multinode cluster: fix waits and joins [#10758](https://github.com/kubernetes/minikube/pull/10758)
+* hyperkit: fix hyperkit-vpnkit-sock setting [#10631](https://github.com/kubernetes/minikube/pull/10631)
+
+Version changes:
+
+* Bump github.com/cheggaaa/pb/v3 from 3.0.6 to 3.0.7 [#10990](https://github.com/kubernetes/minikube/pull/10990)
+* upgrade spinner library v1.12.6 to v1.12.7 [#10983](https://github.com/kubernetes/minikube/pull/10983)
+* BuildKit 0.8.2 [#10648](https://github.com/kubernetes/minikube/pull/10648)
+* Bump google.golang.org/api from 0.40.0 to 0.43.0 [#10945](https://github.com/kubernetes/minikube/pull/10945)
+* bump storage provisioner image [#10951](https://github.com/kubernetes/minikube/pull/10951)
+* Bump golang.org/x/text from 0.3.4 to 0.3.5 [#10946](https://github.com/kubernetes/minikube/pull/10946)
+* Addon: bump metrics-server to v0.4.2 [#10950](https://github.com/kubernetes/minikube/pull/10950)
+* k8s libs: upgrade to v0.20.5 [#10683](https://github.com/kubernetes/minikube/pull/10683)
+* ISO Upgrade Docker, from 20.10.3 to 20.10.4 [#10647](https://github.com/kubernetes/minikube/pull/10647)
+* Addon: bump csi-hostpath-driver to v1.6.0 [#10798](https://github.com/kubernetes/minikube/pull/10798)
+* Bump github.com/otiai10/copy from 1.0.2 to 1.5.0 [#10891](https://github.com/kubernetes/minikube/pull/10891)
+* Bump github.com/shirou/gopsutil/v3 from 3.21.1 to 3.21.2 [#10893](https://github.com/kubernetes/minikube/pull/10893)
+* Upgrade ingress addon files according to upstream(ingress-nginx v0.44.0) [#10879](https://github.com/kubernetes/minikube/pull/10879)
+* Bump k8s.io/klog/v2 from 2.4.0 to 2.8.0 [#10894](https://github.com/kubernetes/minikube/pull/10894)
+* Bump k8s.io/client-go from 0.18.8 to 0.20.5 [#10890](https://github.com/kubernetes/minikube/pull/10890)
+* Bump github.com/cheggaaa/pb/v3 from 3.0.1 to 3.0.6 [#10828](https://github.com/kubernetes/minikube/pull/10828)
+* Bump golang.org/x/mod from 0.4.1 to 0.4.2 [#10829](https://github.com/kubernetes/minikube/pull/10829)
+* Bump github.com/google/go-cmp from 0.5.4 to 0.5.5 [#10747](https://github.com/kubernetes/minikube/pull/10747)
+* Update go-container registry to 0.4.1 + patch [#10731](https://github.com/kubernetes/minikube/pull/10731)
```

Comment:
for Version changes, lets remove all the Go Mod version changes

only version changes that affect user will need to be included

for example Crictl version or Docker Version or Pdoman version

Reply:
done